server/etcdmain/help.go

```diff
@@ -81,6 +81,8 @@ Member:
     Maximum number of operations permitted in a transaction.
   --max-request-bytes '1572864'
     Maximum client request size in bytes the server will accept.
+  --max-concurrent-streams 'math.MaxUint32'
+    Maximum concurrent streams that a client can open at a time.
```

Comment:
nit, let's keep it consistent  (`each` vs `a`) if you modify the PR. @ahrtr

Reply:
thx & updated.